js/libs/form.js

```diff
@@ -416,7 +416,11 @@ Fliplet.Widget.instance('form-builder', function(data) {
             $vm.fields = getFields();
             $vm.isLoading = false;
           }).catch(function (err) {
-            $vm.error = err.message || err.description || err;
+            var error = Fliplet.parseError(err);
+            $vm.error = error;
+            $vm.isLoading = false;
+
+            Fliplet.UI.Toast.error(error);
```

Comment:
```suggestion
            Fliplet.UI.Toast.error(error, {
              message: defaultMessage
            });
```

Without a message, it'll say "Unexpected error" with a "Details" link to learn more. Is that suitable for this scenario or is there a more appropriate message that can be used instead of "Unexpected error"?

Reply:
@tonytlwu thanks! fixed